configurator/src/main.rs

```diff
@@ -20,6 +26,14 @@ struct Cli {
 
 #[derive(Subcommand, Debug)]
 enum Commands {
+    /// Get the difference of two pictures (mostly for demo purposes)
+    Difference {
+        #[arg(short, long, num_args = 2, value_name = "JPG_PATH")]
+        pictures: Vec<PathBuf>,
```

Comment:
Don't like the `Vec` here, won't `[PathBuf; 2]` work?

Reply:
That was the first thing I've tried, but unfortunately both array and tuple won't work without a custom parser, which IMO would just complicate the code without much benefit. See:
https://github.com/clap-rs/clap/issues/1682
https://github.com/clap-rs/clap/issues/1717
Also keep in mind that `num_args = 2` makes sure that there will be exactly two paths provided.

Reply:
That sucks. At least add a comment in line 181 to somehow mark that this is guaranteed to have exactly 2 elements